luatest/server.lua

```diff
@@ -371,14 +371,21 @@ function Server:restart(params, opts)
     log.debug('Restarted server PID: ' .. self.process.pid)
 end
 
--- Save server artifacts by copying the working directory.
--- Throws an error when the copying is not successful.
+--- Save server artifacts by copying the working directory.
+-- The save logic will only work once to avoid overwriting the artifacts directory.
+-- If an error occurred, then server artifacts path will be replaced follow string:
```

Comment:
`then the server artifacts path will be replaced by the following string`

Reply:
Fixed.